Sources/Commands/UserToolchain.swift

```diff
@@ -34,10 +35,16 @@ struct UserToolchain: Toolchain {
 
     init() throws {
         do {
+
             SWIFT_EXEC = getenv("SWIFT_EXEC")
                 // use the swiftc installed alongside ourselves
                 ?? Path.join(Process.arguments[0], "../swiftc").abspath
 
+            lldb = getenv("LLDB")
+                // use the lldb installed alongside ourselves
+                ?? Path.join(Process.arguments[0], "../lldb").abspath
```

Comment:
We only have path for `swiftc` and not `swift` in `Toolchain`. So a path needs to be added either for invoking REPL.

I looked at how swift is launching repl, it's nothing but `lldb --repl`. So I added path for `lldb` directly.